downstream/assemblies/hub/assembly-configure-hub-primary.adoc

```diff
@@ -6,7 +6,7 @@ ifdef::context[:parent-context: {context}]
 :context: configure-hub
 
 [role="_abstract"]
-Configure Red Hat {HubName} as your primary source of content to access Ansible Certified Content Collections. You can configure {HubName} in the command-line interface (CLI) or the web console.
+to access Ansible Certified Content Collections, configure Red Hat {HubName} as your primary source of content. You can configure {HubName} in the command-line interface (CLI) or the web console.
```

Comment:
I think this is the beginning of the sentence--if so, "to" should be capitalized.

Reply:
Good catch! Fixed.